src/client.ts

```diff
@@ -389,9 +391,20 @@ export class Client {
 
     const instrumentations = this.opentelemetryInstrumentations()
 
+    // Metrics initialization
+    let metricReader
+    if (this.config.data["enableOpentelemetryHttp"]) {
+      metricReader = new PeriodicExportingMetricReader({
+        exporter: new OTLPMetricExporter({
+          url: "http://localhost:8099/v1/metrics"
```

Comment:
It could be useful to implement the [`opentelemetry_port` option](https://docs.appsignal.com/python/configuration/options.html#option-opentelemetry_port) for this, so it is customizable, but it's not required.